jetty-http2/http2-hpack/src/main/java/org/eclipse/jetty/http2/hpack/HpackEncoder.java

```diff
@@ -265,7 +265,7 @@ else if (metadata.isResponse())
             // Check size
             if (_maxHeaderListSize > 0 && _headerListSize > _maxHeaderListSize)
             {
-                LOG.warn("Header list size too large {} > {} for {}", _headerListSize, _maxHeaderListSize);
+                LOG.warn("Header list size too large {} > {} for {}", _headerListSize, _maxHeaderListSize, metadata);
                 if (LOG.isDebugEnabled())
```

Comment:
It looks like the intent was only to output the metadata if debug was enabled, so how about this instead:

```suggestion
                if (LOG.isDebugEnabled())
                   LOG.warn("Header list size too large {} > {} metadata={}", _headerListSize, _maxHeaderListSize, metadata);
                else
                    LOG.warn("Header list size too large {} > {}", _headerListSize, _maxHeaderListSize);
```